content/software-security/compliance/cmmc-2/cmmc-documentation.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Documentation Requirements for CMMC 2.0"
+description: "Documentation Requirements for CMMC 2.0"
+lead: "Learn about the different documentation requirements for each of CMMC 2.0's maturity levels. "
+type: "article"
+date: 2024-08-09T19:10:09+00:00
+lastmod: 2024-08-09T19:10:09+00:00
+contributors: ["Erin Glass"]
+draft: false
+tags: ["compliance", "CMMC 2.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "cmmc-2"
+weight: 004
+toc: true
+---
+
+The requirements for documentation under Cybersecurity Maturity Model Certification (CMMC) 2.0 vary by maturity level, with higher levels requiring more extensive documentation to demonstrate compliance. Here is a breakdown of the documentation requirements by maturity level:
+
+## Level 1 Documentation: Foundational
+
+At Level 1, the documentation requirements are minimal, focusing on basic cyber hygiene through the implementation of 17 foundational cybersecurity practices. The purpose is to establish essential protections without the need for extensive documentation.
+
+For example, organizations may maintain basic policies and procedures for access control, media protection, and physical security, along with records of security awareness training. The emphasis at this level is on demonstrating that these fundamental practices are in place, rather than producing detailed documentation, as required in higher levels.
+
+## Level 2 and 3 Documentation: Advanced and Expert
+
+At Level 2, the documentation requirements are moderate, reflecting the need for intermediate cyber hygiene and addressing a subset of the NIST SP 800-171 requirements. Organizations must maintain a System Security Plan (SSP) that outlines security strategies and vulnerability assessment and remediation plans. They must also create a Plan of Action and Milestones (POA&M) addressing any aspects of the organization which are note yet implemented. Other documentation requirements may include audit logs, incident response reports, inventory of the organization’s systems, location of CUI in the organization’s environment, and other documents related to the implementation and management of cybersecurity practices.
```

Comment:
This is a lot of info for a paragraph. Depending on how much of an actual list this is (versus just a subset of things being used as examples), it could be useful to put items in an unordered list where readers can quickly scan rather than read and parse for commas and new entries.